index.html

```diff
@@ -1,214 +1 @@
-                <div class="bloc-contact">
-                    <h3 class="nom-du-contact">Laura De Berin</h3>
-                    <p>Visual SFX Supervisor<br>
-                        Réalisatrice<br>
-                        06 xx xx xx xx</p>
-                </div>
-            </div>
-
-        </section>
-
-        <!--Section 6 A propos-->
-        <section id="about" class="bloc-sider">
-            <div class="bandeau-title">
-                <h2 class="side-title">À propos de moi</h2>
-            </div>
-            <div class="bloc-texte-side">
-                <p>J'ai 33, j'ai travaillé dans le domaine de la télévision pendant 9 ans après avoir suivi des études
-                    d'arts et d'audiovisuel.
-                    Ayant toujours eu un intérêt et des contacts dans les domaines du graphisme, du jeu vidéo et du
-                    développement numérique, j'ai au fil de rencontres
-                    choisi de me réorienter vers le développement front-end et j'ai ainsi démarré une formation
-                    d'intégrateur web sur OpenClassroom.</p>
-                <p>Je suis passionné de musiques et de design sonore. </p>
-            </div>
-
-        </section>
-
-        <!--Section 7 Compétences-->
-        <section id="skills" class="bloc-sider">
-            <div class="bandeau-title">
-                <h2 class="side-title">Mes Compétences</h2>
-            </div>
-            <div class="bloc-texte-side">
-                <h4 class="skill-group-title">Développement</h4>
-                <ul>
-                    <li>HTML 5, CSS 3</li>
-                    <li>BootStrap 4</li>
-                    <li>GitHub</li>
-                    <li>Intégrer une maquette</li>
-                    <li>Standard W3C</li>
-                    <li>Référencement</li>
-                    <li>Bases de Java, C, Python</li>
-                </ul>
-                <h4 class="skill-group-title">Logiciels</h4>
-                <ul>
-                    <li>Adobe Photoshop</li>
-                    <li>Adobe Illustrator</li>
-                    <li>Adobe XD</li>
-                    <li>Inkscape, Krita</li>
-                    <li>Reaper (Mixage Audio)</li>
-                    <li>Reason (MAO)</li>
-                    <li>Da Vinci, Nuke (Montage Vidéo)</li>
-                </ul>
-                <h4 class="skill-group-title">Audiovisuel</h4>
-                <ul>
-                    <li>Mixage et Mastering tv</li>
-                    <li>Console de mixage</li>
-                    <li>Placement des micros</li>
-                    <li>Conformité des signaux</li>
-                </ul>
-            </div>
-
-        </section>
-
-        <!--Section 8 Loisirs-->
-        <section id="loisirs" class="bloc-sider">
-            <div class="bandeau-title">
-                <h2 class="side-title">Mes Loisirs</h2>
-            </div>
-            <div class="bloc-icon-side">
-                <i class="fas fa-compact-disc"></i>

```

Comment:
Que se passe-t-il pour les moteurs de recherche ou les lecteurs d'écran ici? Tu n'as aucun contenu textuel sur cette section.
Jete un oeil à ce guide de Font Awesome https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility